pipeline/pipeline.go

```diff
@@ -140,6 +142,8 @@ type Settings struct {
 func New(name string, settings *Settings, registry *prometheus.Registry) *Pipeline {
 	metricCtl := metric.New("pipeline_"+name, registry)
 
+	lg := logger.Instance.Named(name)
```

Comment:
nit: `lg` should be passed to `Pipeline.logger` instead of creating new logger with the same name

Reply:
done